### PR TITLE
added system requirements

### PR DESCRIPTION
**Pull request by tpltnt**
Hi there,

I added some information about the system requirements to the readme file (because they weren't clear to me). No functional/code changes.
Cheers,
tpltnt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +9,12 @@ Please contribute!  Make sure to review the [pull request checklist](/CONTRIBUTI
 
 If you find rr useful, please [add a testimonial](https://github.com/mozilla/rr/wiki/Testimonials).
 
+# systems requirements
+* Intel chip with [Nehalem](https://en.wikipedia.org/wiki/Nehalem_%28microarchitecture%29) (2010) or later microarchitecture
+
+ * VM with perf counter virtualization is ok
+ * userspace: x86
```
Comment (khuey) [inline README.md]: We support 64 bit userspace now.

File: README.md [hunk 2]
```diff
@@ -8,3 +9,12 @@ Please contribute!  Make sure to review the [pull request checklist](/CONTRIBUTI
 
 If you find rr useful, please [add a testimonial](https://github.com/mozilla/rr/wiki/Testimonials).
 
+# systems requirements
+* Intel chip with [Nehalem](https://en.wikipedia.org/wiki/Nehalem_%28microarchitecture%29) (2010) or later microarchitecture
```
Comment (khuey) [inline README.md]: system requirements
Intel CPU

File: README.md (hunk 1 above)
Comment (tpltnt) [inline README.md]: Ah, cool. My main source was [this presentation](https://mozilla.github.io/rr/rr.html) and it didn't mention 64bit support. I updated the readme. Thank you for pointing out the 64bit support.

Comment (khuey) [inline README.md]: Yeah, the presentation is ancient :/

**Comment by rocallahan** [comment]
I think rr runs on Westmere, which is pre-Nehalem. Why did you put Nehalem?

To simplify things I'd just make the mininum kernel version 3.5 and assume seccomp-bpf is required.

**Comment by rocallahan** [comment]
@khuey tells me that Nehalem is correct.

**Comment by rocallahan** [comment]
rr is known to work with 32-bit kernels (I test that before release).

**Comment by rocallahan** [comment]
Merged with changes.

**Comment by Keno** [comment]
> To simplify things I'd just make the mininum kernel version 3.5 and assume seccomp-bpf is required.

Unfortunately, I'm currently learning that some distributions (RHEL is the one I have here), have newer kernels but force-disable seccomp-bpf at kernel build time, so maybe it would be good to retain the ability to work without it.

**Comment by rocallahan** [comment]
Hmm. Unfortunately I think we already require seccomp-bpf even if syscall buffering is disabled, because we depend on it to generate the right ptrace traps. See "Use a dummy filter" in `Task.cc`.

**Comment by rocallahan** [comment]
(And that's been true for quite a long time.)

**Pull request closed without merging**